tests/keras/engine/test_topology.py

```diff
@@ -76,6 +76,35 @@ def test_trainable_weights():
     assert model.non_trainable_weights == weights
 
 
+def test_compute_mask():
+    model = Sequential()
+    model.add(Dense(1, input_dim=2))
+    assert model.layers[0].supports_masking is True
+    assert model.layers[0].compute_mask([model.input], [0., 1.]) == [0., 1.]
+
+    model = Sequential()
```

Comment:
I agree. This test has been divided into as you suggested: `test_valid_compute_mask` and `test_invalid_compute_mask`.